modules/ROOT/pages/cassandra/cassandra-xml-maven.adoc

```diff
@@ -0,0 +1,59 @@
+= XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+XML and POM file configuration information and XML-related usage topics.
```

Comment:
This topic does not have enough context for me to understand. Is this all and only what I have to do? How is this different from the XML example I saw in Use Cases? Why didn't Use Cases mention maven/pom?

Reply:
Feedback for Meera?